src/main/java/duke/UnmarkCommand.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+/**
+ * This class handles all commands related to unmarking tasks
+ * and inherits from the Command class.
+ */
+public class UnmarkCommand extends Command {
+    private int index;
+
+    /**
+     * Constructor for the UnmarkCommand class.
+     *
+     * @param index to indicate the task to be unmarked.
+     */
+    public UnmarkCommand(int index) {
+        this.index = index;
+    }
+
+    /**
+     * {@inheritDoc}
+     * Unmarks a task from the tasklist.
+     */
+    @Override
+    public String execute(TaskList taskList, Ui ui, Storage storage) {
+
```

Comment:
Perhaps you could remove this empty new line if it is unnecessary? I noticed the same issue in several other places too.